src/hooks/useAllowance.ts

```diff
@@ -16,8 +16,8 @@ export const useLotteryAllowance = () => {
 
   useEffect(() => {
     const fetchAllowance = async () => {
-      const res = await cakeContract.methods.allowance(account, getLotteryAddress()).call()
-      setAllowance(new BigNumber(res))
+      const res = await cakeContract.allowance(account, getLotteryAddress())
+      setAllowance(new BigNumber(res.toString()))
```

Comment:
Hey Doge - just checking, as this is relevant for the multicall implementation I'm finishing up in the lottery. Is it OK to go from an ethers.BigNumber (i.e. what I think will be returned here in `res`), and then construct a new BigNumber.js BN, by passing in the ethers.BigNumber & calling `.toString()` on it? Will we look to migrate this in future too?

Reply:
yes, the string representation can be passed in the contructor

Reply:
Sure - I more mean in terms of working with multiple BigNumber libraries - is that going to be the strategy?

Ethers response uses BN.js
The response is passed into a BigNumber.js constructor
Thereafter in the FE for calculations etc, BigNumber.js is used?

Reply:
this is the short term strategy, until we get rid of BigNumber.js